### ISSUE TITLE
CC set to gcc is not very portable

### ISSUE THREAD

**Issue by shmish111** [JakobBruenker/curses-idris#3]
Is there a good reason for setting `CC` to `gcc` in the Makefile? It means I cannot build on OSX using nix :-(

**Comment by JakobBruenker** [comment]
There isn't. I'm not sure how to change it to make it portable though - or would just removing that line suffice? Worst case scenario, you could just fork it and change the Makefile in your fork.

(Also note that I haven't worked on this repository in years, I'm not sure whether it actually still works with the current Idris version.)

**Comment by shmish111** [comment]
I haven't tested it, I will have a go. I'm just going through idris packages that are in nixpkgs but don't work. Would you say this package is still useful to others? If not it could be removed.

**Comment by JakobBruenker** [comment]
Yeah, given that it's currently unmaintained, removing it would probably be for the best - I didn't know it was in nixpkgs. Was that an offer to take care of it or should I do it? :)

**Comment by shmish111** [comment]
I would appreciate it if you could make a PR in nixpkgs. Are you still interested in Idris these days? I'm trying to push for some progress in package management and fundamental libraries in 2019 to make Idris a bit more useable.

**Comment by JakobBruenker** [comment]
Will do. I'm still interested, but I haven't really had time to keep up with the development or actually do things with it. That sounds like a good idea though.
